NSS-Day-Cohort-42/rare-api-fakes-news
# working def update and def create #209

#209 Working def update and def create

## Description
Adds functionality to postman for def update and create

## Tickets
_My PR references these tickets (epic and task tickets)_
#209

## How do I access this branch?

1. ```git fetch --all```
1. ```git checkout ``` 
1. ```python manage.py runserver``` 

​

​
## Types of changes
- [x ] New feature (non-breaking change which adds functionality)

## Testing
- [ ] Tested in Postman
- [ ] Tested with Client


``` if necessary, postman code here ```


1.Check def create by switching to POST and copy pasting this:
"{
    "id": 3,
    "post_id":2,
    "content": "This is the content!!!!",
    "subject": "This is the subject!!!!",
    "created_on": "2020-11-09T13:19:37Z"
}"​
In body make sure to hit the bulletpoint raw and switch to json in dropdown menu
2. Check and see if new comment was created in GET
3. Check def update by going to http://localhost:8000/comments/1 and use PUT and change something in the comment format above.
4.Check list again to make sure it was edited..

## Checklist for Reviewer
- [ ] check it out
- [ ] test all edge cases listed and more if you think of them (postman or browser)
- [ ] request changes/approve
- [ ] review code to make sure it fits coding style of project
